### PR TITLE
add repo_debian_base

### PR DESCRIPTION
**Pull request by lfchris**

### CODE REVIEW

File: roles/repo_debian_base/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Ansible Role linuxfabrik.lfops.repo_debian_base
+
+This role deploys the basic Debian repositories, which can be used to set a custom mirror server.
+
+Runs on
+
+* Debian11 (and compatible)
```
Comment (NavidSassan) [inline roles/repo_debian_base/README.md:7]:
```suggestion
* Debian 11 (and compatible)
```

Comment (NavidSassan) [inline roles/repo_debian_base/README.md:7]: did you also test it with debian 10? asking because of the `vars/Debian10.yml`

**Review by NavidSassan — APPROVED** [review]
looks good

File: roles/repo_debian_base/README.md (hunk 1 above)
Comment (lfchris) [reply roles/repo_debian_base/README.md]: Nein noch nicht. Kann ich noch kurz machen

**Pull request merged**